feat(squidgame): adding new theme called squidgame

sandrahlee opened the pull request:
I just love...squidgame so much.

sandrahlee committed:
feat(squidgame): adding new theme called squidgame

sandrahlee commented:
![image](https://user-images.githubusercontent.com/54752230/138791524-82439d83-ea00-4b34-9bf8-2a35adc5305b.png)
![image](https://user-images.githubusercontent.com/54752230/138791539-fe63603d-81b7-4d9d-9778-78493ebec4fc.png)

jakergrossman reviewed (changes requested):
👍 If `img/squidgameredo.png` isn't going to be used (I don't think it should be), it should be omitted from the pull request :smile:.

sandrahlee committed:
removing the img that is being unused

sandrahlee commented:
> 👍 If `img/squidgameredo.png` isn't going to be used (I don't think it should be), it should be omitted from the pull request 😄.

Totally forgot I have added that img. It is now removed.
